operator/cmd/mesh/testdata/manifest-generate/output/gateways.golden.yaml

```diff
@@ -203,7 +203,7 @@ spec:
             cpu: 2000m
             memory: 1024Mi
           requests:
-            cpu: 111m
+            cpu: 999m
```

Comment:
This is suspicious. Is this a bug fix or a bug?

Reply:
it's a new test case. the setting would be 111m using the overlay but the --set flag overrides it to 999m. it simultaneously tests the set flag and overlay priority.